external/anomaly/anomaly_classification/openvino.py

```diff
@@ -133,51 +133,42 @@ def optimize(
         with tempfile.TemporaryDirectory() as tempdir:
             xml_path = os.path.join(tempdir, "model.xml")
             bin_path = os.path.join(tempdir, "model.bin")
-            with open(xml_path, "wb") as xml_file:
-                xml_file.write(self.task_environment.model.get_data("openvino.xml"))
-            with open(bin_path, "wb") as bin_file:
-                bin_file.write(self.task_environment.model.get_data("openvino.bin"))
+            with open(xml_path, "wb") as weight_file:
+                weight_file.write(self.task_environment.model.get_data("openvino.xml"))
+            with open(bin_path, "wb") as weight_file:
+                weight_file.write(self.task_environment.model.get_data("openvino.bin"))
 
-            model_config = ADDict({"model_name": "openvino_model", "model": xml_path, "weights": bin_path})
```

Comment:
I'm looking at the documentation of `addict`, where a dictionary is created by `Dict` itself. If we don't use the native `Dict` in the module, I think we could also do the same. It's because when reading the code `ADDict` could  distracts.